csharp/src/Ice/OutputStream.cs

```diff
@@ -1559,6 +1562,18 @@ public BitSequence WriteBitSequence(int bitSize)
             }
         }
 
+        internal static void WriteEncapsSize(int size, Span<byte> data, Encoding encoding)
+        {
+            if (encoding == Encoding.V2_0)
+            {
+                WriteSize20(size, data);
```

Comment:
WriteSize20 is a "temporary helper" that writes the size on 4 bytes.

Reply:
Fixed